Makefile

```diff
@@ -82,16 +82,27 @@ endif
 checkout:
 	./scripts/checkout.sh
 
+#This could currently break if not using COSMOS in the config.
 ci-launch:
 	@export SYSTEM_TEST_FILE_PATH=$(SYSTEM_TEST_FILE_PATH) && \
 	./scripts/ci_launch.sh && \
+	./scripts/system_tests.sh && \
 	./scripts/stop.sh
 
+#This could currently break if not using COSMOS in the config.
 system-tests:
 	@export SYSTEM_TEST_FILE_PATH=../..$(SYSTEM_TEST_FILE_PATH) && \
 	./cfg/build/launch.sh && \
 	./scripts/system_tests.sh
 
+#Be sure that your nos3-mission.xml has been set to YAMCS
+yamcs-operator:
+	@export SYSTEM_TEST_FILE_PATH=$(SYSTEM_TEST_FILE_PATH) && \
+	./scripts/ci_launch.sh --use-yamcs
```

Comment:
May have a problem using an additional flag here in that you likely need the build step for that particular ground software as well. It may be included as part of the script now, but as that changes now we'd need to update those scripts and this ci_launch.sh one.